### ISSUE TITLE
Error in Lwt_stream.​bounded_push documentation

### ISSUE THREAD

**Issue by Khady** [ocsigen/ocsigen.org#25]
In multiple function description the string `Error a_api: "bounded_push" is not a valid module name` is used instead of a correct type/value.

https://ocsigen.org/lwt/2.5.2/api/Lwt_stream.bounded_push-c

**Comment by aantron** [comment]
Ok, while debugging this, I found:

- This occurs on references to methods (obviously).
- `ocsigen.org`'s `a_api` wiki extension expects method references to have the syntax `method Path.class#method` (note the `#`) [[1]], [[2]].
- `wikidoc` generates syntax `method Path.class.method` (note the `.`) [[3]].
- `ocsigen.org` therefore treats the `class` part as an attempt at a module name, resulting in the observed messages.

I can submit a PR, but to which project? IMO since `wikidoc` simply outputs the path that `ocamldoc` provides, we should change `ocsigen.org` to accept that syntax.

[1]: https://github.com/ocsigen/ocsigen.org/blob/2b47c5702bf19de170649003e461011b91833cdb/src/site_doc_link.ml#L360
[2]: https://github.com/ocsigen/ocsigen.org/blob/2b47c5702bf19de170649003e461011b91833cdb/src/site_doc_link.ml#L408
[3]: https://github.com/ocsigen/wikidoc/blob/f49bd8188a21628e307d891bd3b46ac9ca8f7e8f/src/odoc_wiki/odoc_wiki.ml#L359

**Comment by Drup** [comment]
@aantron I would say it's better to fix wikidoc here, but honestly, do what is the easiest. ;)

**Comment by aantron** [comment]
Why?

It's easiest for me to fix `wikidoc`, because I can run it locally to test my work. But the fix would be replacing the last `.` by `#`, which IMO is the wrong thing to do for the long term. It's better to just be consistent about format. Since this error has not been reported before (apparently), I don't think I'll break anything by messing with `ocsigen.org`. So, as long as it's possible to deploy the fix with relative ease, I prefer to fix here. I'll submit a PR.

**Comment by Drup** [comment]
> So, as long as it's possible to deploy the fix with relative ease

Yes, that's precisely what I'm worried about ... @vasilisp, @balat ?
